### ISSUE TITLE
[Accessibility] Inspect or AccessibilityInsights tool's rectangle can't focus on the entire PrintPreviewControl

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#8892]
### .NET version

.NET SDK 8.0.100-preview.4.23172.16

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, it repro in .NET 8.0 & 7.0, cannot repro in .NET 6.0 & 5.0 & 3.1.

### Issue description

Inspect or AccessibilityInsights tool's rectangle can't focus on the entire PrintPreviewControl.
**Actual result:**
AccessibilityInsights:
![core=ppc](https://user-images.githubusercontent.com/94418985/227127203-6837b313-65f7-4913-a1f5-14dd3ae58401.png)

Inspect:
![inspect=ppc](https://user-images.githubusercontent.com/94418985/227127242-478beda5-55b4-4c16-9d50-5a9dd4519461.png)

**Expected result:**
AccessibilityInsights:
![frame=ppc](https://user-images.githubusercontent.com/94418985/227127187-9194ea3b-2c77-473e-a0de-0df9ab49230c.png)

Inspect:
![inspect=ppc=fra,](https://user-images.githubusercontent.com/94418985/227127285-cfdcb424-00fa-4c66-901c-27e93a71a3b4.png)

### Steps to reproduce

1. Create a winforms core project with a PrintPreviewControl.
2. Build and run.
3. Launch Inspect or Accessibility Insight tool.
4. Focus on the entire PrintPreviewControl.

**More info:**
1. When set PrintPreviewControl TabStop property true or false, all can repro this issue.

**Comment by merriemcgaw** [comment]
Interesting! I wonder what changed. @MelonWang1 is it possible to test on an earlier version of Windows? Maybe there was a Windows change here.

**Comment by MelonWang1** [comment]
@merriemcgaw Verify this issue in VS latest release version 17.5.5 on win8.1 machine and VS version 17.7.0 Preview 2.0 [33703.476.main] on win10 rs1 machine, it still repro in .NET 7.0, cannot repro in .NET 6.0.

**Comment by dmitrii-drobotov** [comment]
It didn't confirm it, but it was probably introduced in https://github.com/dotnet/winforms/pull/6600, where we implemented accessible object for the control.

**Comment by Tanya-Solyanik** [comment]
@v-elnovikova - since PrintPreview is a native control, would it make sense to defer to the native accessible object to access bounds?

**Comment by v-elnovikova** [comment]
> @v-elnovikova - since PrintPreview is a native control, would it make sense to defer to the native accessible object to access bounds?

PrintPreviewControl isn't native actually. I added overridden `Bounds` property, it now returns the bounds of a control  - and it worked.

**Comment by Nora-Zhou01** [comment]
Verified this on .NET8.0.100-preview.5.23274.2, issue was fixed, Inspect or AccessibilityInsights tool's rectangle can be focus on the entire PrintPreviewControl.
Inspect:
![屏幕截图 2023-05-24 225736](https://github.com/dotnet/winforms/assets/104609169/4a594dd0-6fe9-49c6-94bb-825e13c102c7)
AccessibilityInsights:
![屏幕截图 2023-05-24 225615](https://github.com/dotnet/winforms/assets/104609169/2a3015c8-709d-4134-8270-908021e46ae2)

**Comment by Ashley-Li** [comment]
Verified this on .NET 8.0 Preview 5.0 TP build: 8.0.100-preview.5.23276.4, issue was fixed as same as above result.

**Comment by Amy-Li03** [comment]
Verified this on .NET 8.0 TP build: 8.0.100-preview.5.23276.4, issue was fixed, test result is same as above screenshot.
